iOS_SDK/OneSignalSDK/Source/OSMigrationController.m

```diff
@@ -56,21 +56,25 @@ - (void)migrateToVersion_02_14_00_AndGreater {
     let uniqueCacheOutcomeVersion = 21403;
     let sdkVersion = [OneSignalUserDefaults.initShared getSavedIntegerForKey:OSUD_CACHED_SDK_VERSION defaultValue:0];
     if (sdkVersion < influenceVersion) {
-        [OneSignal onesignal_Log:ONE_S_LL_DEBUG message:[NSString stringWithFormat:@"Migrating OSIndirectNotification from version: %d", sdkVersion]];
+        [OneSignal onesignal_Log:ONE_S_LL_DEBUG message:[NSString stringWithFormat:@"Migrating OSIndirectNotification from version: %ld", sdkVersion]];
```

Comment:
Changing it to %ld is correct but it is getting a warning because it thinks sdkVersion should be an int. Instead of `let sdkVersion` lets do `long sdkVersion` to get rid of the warnings.